subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/buildcache/TaskOutputCachingJavaPerformanceTest.groovy

```diff
@@ -117,7 +118,8 @@ class TaskOutputCachingJavaPerformanceTest extends AbstractTaskOutputCachingPerf
         result.assertCurrentVersionHasNotRegressed()
 
         where:
-        [testProject, tasks] << scenarios
+        testProject = SMALL_JAVA_MULTI_PROJECT
```

Comment:
I guess you didn't want to commit these changes.

Reply:
Good catch, reverted.